### ISSUE TITLE
MAC address resets when connecting with NetworkManager

### ISSUE THREAD

**Issue by smclt30p** [alobbs/macchanger#24]
Card: Intel PRO/Wireless 4965 AGN
Driver: iwl4965

When connecting with NetworkManager to a WPA/WPA2 AP, the MAC address resets itself. Does not happen when using wpa_supplicant manually.

**Comment by thoger** [comment]
Looks like a dupe of issue #22.

**Comment by smclt30p** [comment]
Yup, It is a NetworkManager issue. Someone, feel free close this issue.
